packages/website/src/pages/index.tsx

```diff
@@ -161,7 +161,7 @@ function Home(): JSX.Element {
             key={idx}
             className={clsx(
               styles.features,
-              idx % 2 == 1 ? styles.lightBackground : '',
+              idx % 2 === 1 ? styles.lightBackground : '',
```

Comment:
Would you be open to splitting out the ESLint changes into their own PR, to make it more clear what's changing for them? This PR is pretty big.

Reply:
Yeah I was just hacking on things to make it work then I noticed these files weren't being linted and fixed that.

I'll fix it in master - there are more files that need to be fixed anyways!